src/oge/load_data.py

```diff
@@ -1224,6 +1202,9 @@ def load_emissions_controls_eia923(year: int) -> pd.DataFrame:
         emissions_controls_eia923["report_date"] = emissions_controls_eia923[
             "report_date"
         ].astype("datetime64[s]")
+        emissions_controls_eia923["equipment_tech_description"] = (
+            emissions_controls_eia923["equipment_tech_description"].str.lower()
+        )
     else:
         logger.warning(
             "Emissions control data prior to 2012 has not been integrated into the data pipeline."
```

Comment:
Is that 2012 or 2008?

Reply:
PUDL has integrated data from 2008 onward, but their version of the table does not currently include columns for hours_in_service and ozone_season_nox_emission_rate. See: https://github.com/catalyst-cooperative/pudl/issues/3689. 

We are still manually loading the EIA-923 table from a raw download, and we have only integrated back to 2012 so far.